app/lib/beta_test_data_generator.rb

```diff
@@ -0,0 +1,96 @@
+    "Katie", "Jade", "Catherine", "Diana", "Aubrey", "Mya", "Amy", "Briana", "Sophie", "Gabriela", "Breanna", "Gianna",
+    "Kennedy", "Gracie", "Peyton", "Adriana", "Christina", "Courtney", "Daniela", "Lydia", "Kathryn", "Valeria", "Layla",
+    "Alexandria", "Natalia", "Angel", "Laura", "Charlotte", "Margaret", "Cheyenne", "Miranda", "Mikayla", "Naomi",
+    "Kelsey", "Payton", "Ana", "Alicia", "Jillian", "Daisy", "Ashlyn", "Sabrina", "Caitlin", "Summer",
+    "Ruby", "Rylee", "Valerie", "Skylar", "Lindsey", "Kelly", "Genesis", "Zoey", "Eva", "Sadie", "Alexia", "Cassidy",
+    "Kylee", "Kendall", "Jordyn", "Kate", "Jayla", "Karen", "Tiffany", "Cassandra", "Juliana", "Reagan", "Caitlyn",
+    "Giselle", "Serenity", "Alondra", "Lucy", "Bianca", "Kiara", "Crystal", "Erica", "Angelica", "Hope", "Chelsea",
+    "Alana", "Liliana", "Brittany", "Camila", "Makenzie", "Lilly", "Veronica", "Abby", "Jazmin", "Adrianna", "Delaney",
+    "Karina", "Ellie", "Jasmin"]
+
+  LAST_NAMES = [
+    "Alfalfa", "Apple", "Apricot", "Artichoke", "Asparagus", "Avocado", "Almond",
+    "Banana", "Bean", "Beans", "Beets", "Blackberry", "Blueberry", "Boysenberry", "Broccoli", "Basil",
+    "Cabbage", "Cantaloupe", "Carrots", "Cassava", "Cauliflower", "Celery", "Chayote", "Cherimoya", "Cherry", "Coconut", "Collards", "Corn", "Cranberry", "Cucumber",
+    "Date", "Dill", "Daikon", "Dandelion Greens", "Delicata", "Dragonfruit",
+    "Eggplant", "Escarole", "Endive", "Epazote",
+    "Fennel", "Fig", "Fenugreek",
+    "Garlic", "Gooseberry", "Grapefruit", "Grape", "Green Bean", "Green Onion", "Guava", "Galangal", "Ginger",
+    "Honeydew", "Habanero", "Hazelnut", "Hibiscus", "Horseradish",
+    "Ilama", "Imbé", "Icaco",
+    "Jicama", "Jujube", "Jalapeño", "Jitomate",
+    "Kale", "Kiwi", "Kohlrabi", "Kumquat",
+    "Leek", "Lemons", "Lettuce", "Lima Bean", "Lime", "Longan", "Loquat", "Lychee", "Lemongrass",
+    "Mandarin", "Mango", "Mulberry", "Mushroom", "Mangosteen", "Marjoram", "Maple", "Matcha", "Mustard", "Mustard-Greens", "Mint",
+    "Nectarines", "Nopales", "Nutmeg", "Nigella",
+    "Okra", "Onion", "Orange", "Oregano",
+    "Papaya", "Parsnip", "Passionfruit", "Peach", "Pear", "Pea", "Peas", "Pepper", "Persimmon", "Pineapple", "Plantain", "Plum", "Pomegranate", "Potato", "Prickly Pear", "Prune", "Pomelo", "Pumpkin", "Prickly Pear",
+    "Quince",
+    "Radicchio", "Radish", "Raisin", "Raspberry", "Rhubarb", "Romaine", "Rutabaga", "Rosemary",
+    "Shallot", "Snow-Pea", "Spinach", "Sprouts", "Squash", "Strawberry", "String Bean", "Sweet-Potato", "Serrano", "Sesame",
+    "Tangelo", "Tangerine", "Tomatillo", "Tomato", "Turnip", "Thyme", "Tea", "Tamarind",
+    "Ube", "Uva",
+    "Vidalia", "Verdolaga", "Vanilla",
+    "Water Chestnut", "Watercress", "Watermelon", "Waxed Bean", "Wasabi", "Walnut",
+    "Xocota", "Ximenia",
+    "Yam", "Yucca",
+    "Zucchini",
+  ]
+
+  STATUSES_AFTER_OPEN = TaxReturnStatus::STATUSES.select { |key, value| value >= 102 }.keys.freeze
```

Comment:
Could use `TaxReturnStatus::STATUSES[:intake_open]` instead of `102`. Not a blocker. Might improve readability.